src/vma/util/vma_list.h

```diff
@@ -251,12 +251,14 @@ class  vma_list_t
 	inline T* get_and_pop_front() {
 		T* list_front = front();
 		pop_front();
+		((list_node<T, offset> *)m_list.head.next)->obj_ptr = NULL;
```

Comment:
Hi @igor-ivanov,
Please consider remove this line and add the following link to [erase()](https://github.com/Mellanox/libvma/blob/master/src/vma/util/vma_list.h#L270) method :
node_obj->obj_ptr = NULL;